src/data/queries/mediaDocument.ts

```diff
@@ -0,0 +1,46 @@
+import { QueryData, QueryFormatter, QueryParams } from 'next-drupal-query'
+import { DrupalJsonApiParams } from 'drupal-jsonapi-params'
+import { drupalClient } from '@/lib/drupal/drupalClient'
+import { MediaDocument } from '@/types/formatted/media'
+import { DrupalMediaDocument } from '@/types/drupal/media'
+
+// Define the query params for fetching node--media_document.
+export const params: QueryParams<null> = () => {
+  return new DrupalJsonApiParams().addInclude(['document'])
+}
+
+// Define the option types for the data loader.
+export type DataOpts = {
+  id: string
+  context?: string
+}
+
+// Implement the data loader.
+export const data: QueryData<DataOpts, DrupalMediaDocument> = async (
+  opts
+): Promise<DrupalMediaDocument> => {
+const entity = await drupalClient.getResource<DrupalMediaDocument>(
+    'media--document',
+    opts.id,
+    {
+      params: params().getQueryObject(),
+    }
+  )
+
+  return entity
+}
+
+export const formatter: QueryFormatter<DrupalMediaDocument, MediaDocument> = (
+  entity: DrupalMediaDocument
+) => {
+  if (!entity) return null
+  return {
+    id: entity.document.id,
+    type: entity.type,
+    links: entity.document?.links,
+    alt: entity.document?.resourceIdObjMeta?.alt,
+    width: entity.document?.resourceIdObjMeta?.width,
+    height: entity.document?.resourceIdObjMeta?.height,
+    title: entity.document?.resourceIdObjMeta?.title,
```

Comment:
all of these would need updated to `entity.field_document.`. I'm also not sure they are the correct values to be formatting to here. 

https://content-build-medc0xjkxm4jmpzxl3tfbcs7qcddsivh.ci.cms.va.gov/jsonapi/media/document/b9ce0eee-538e-48b2-9aeb-6ee5c1294030?include=field_document

surfaces these as the fields of an actual file:
```
        "drupal_internal__fid": 10,
        "langcode": "en",
        "filename": "VA.gov-outreach-library-2019-01-30-jl.pdf",
        "uri": {
          "value": "public://2019-02/VA.gov-outreach-library-2019-01-30-jl.pdf",
          "url": "/sites/default/files/2019-02/VA.gov-outreach-library-2019-01-30-jl.pdf"
        },
        "filemime": "application/pdf",
        "filesize": 332371,
        "status": true,
        "created": "2019-02-07T19:04:22+00:00",
        "changed": "2019-02-07T19:04:32+00:00"
```

We'll likely want `filename`, `uri`, `filemime` (mimetype), `filesize`